### PR TITLE
Customize CMS preview pane

### LINKED ISSUE #14: Create Custom Preview
Creating a preview template and registering it in the CMS will make the preview pane more akin to the final product once a page is published. [Netlify CMS docs for custom previews](https://www.netlifycms.org/docs/customization/)

### PR DESCRIPTION
**Pull request by UtkarshKr007**
This PR will customize the preview pane of any fact to look more akin to it's representation on the live website.

### CODE REVIEW

**Comment by floogulinc** [comment]
It works!
image

File: src/cms/cms.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+const factPageStyles = makeStyles({
+  card: {
+    maxWidth: 520,
+    marginTop: 20,
+    margin: 'auto',
+  },
+});
+
+const FactPagePreview = ({ entry, widgetFor, getAsset }) => {
```
Comment (floogulinc) [inline src/cms/cms.js:15]: If we end up adding additional types of pages we will want to split this into its own file.

File: src/cms/cms.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          className={classes.card}
+          factTitle={entry.getIn(['data','title'])}
+          factImage={factImg}
+          factSource={entry.getIn(['data','source'])}
+          factCategory={entry.getIn(['data','category'])}
+          factHTML={entry.getIn(['data','body'])}
+          />
+    </>
+  )
+}
```
Comment (floogulinc) [inline src/cms/cms.js:45]: If we end up adding additional types of pages we will want to split this into its own file.

**Comment by kklamberty** [comment]
Is this pull request ready for review? It looks like it is still a draft. @UtkarshKr007 ?

**Review by kklamberty — APPROVED** [review]

**Pull request merged**